blueprints.yaml

```
@@ -50,6 +50,31 @@ form:
       accept:
         - image/png
 
+    show_external_links:
+      type: toggle
+      label: Show Extra Links in Navigation Menu
+      highlight: 0
+      default: 0
+      options:
+        1: PLUGIN_ADMIN.YES
+        0: PLUGIN_ADMIN.NO
+      validate:
+        type: bool
+
+    external_links:
+      type: list
+      style: horizontal
+      label: Extra Links
+      description: Additional links to be shown in the navigation menu.
```

We determined that descriptions don't show up on lists, right? If so, let's change this.
```suggestion
      help: Additional links to be shown in the navigation menu.
```